README.md

```diff
@@ -53,6 +53,7 @@ following environment variables:
 | Variable | Default | Explanation |
 | -------- | ------- | ----------- |
 | `PORTUNUS_DEBUG` | `false` | When true, log debug messages to standard error. May cause passwords to be logged. **Do not use in production.** |
+| `PORTUNUS_DOMAIN_NAME` | `portunus` | The domain name used in TOTP token issuer field. |
```

Comment:
What is the significance of this field? Does it appear in the TOTP UI? The documentation should explain what goes into choosing a value for this.

**Edit:** I can see that all of the test cases use `portunus.example.com`. Should we have this just match the domain name that's implied in `PORTUNUS_LDAP_SUFFIX`? In that case, could we just derive it from `PORTUNUS_LDAP_SUFFIX` (e.g. `dc=example,dc=com -> example.com`), or do we need to be able to have a different value here?